docs/source/gbd2019_models/risk_exposures/sbp/index.rst

```diff
@@ -84,68 +84,59 @@ Restrictions
      - Value
      - Notes
    * - Male only
-     -
+     - False
      -
    * - Female only
+     - False
      -
+   * - YLD only
+     - False
      -
-   * - Age group start
-     -
+   * - YLL only
+     - False
      -
+   * - Age group start
+     - 10
+     - [25, 29 years)
    * - Age group end
-     -
-     -
-
-..	todo::
-
-	Determine if there's something analogous to "YLL/YLD only" for this section
+     - 235
+     - [95, 125 years)
 
 Assumptions and Limitations
 +++++++++++++++++++++++++++
 
-Describe the clinical and mathematical assumptions made for this cause model,
-and the limitations these assumptions impose on the applicability of the
-model.
-
-Risk Exposure Model Diagram
-++++++++++++++++++++++
+The quantity of interest is exposure to the mean blood pressure level regardless of whether that level is naturally occurring or occurs via use of medication; we assume full reversibility of risk and do not account for duration of exposure to elevated SBP. 
 
-Include diagram of Vivarium risk exposure model.
+Data Description
+++++++++++++++++
 
-Data Description Tables
-+++++++++++++++++++++++
+The rei_id for SBP is 107
 
-As of 02/10/2020: follow the template created by Ali for Iron Deficiency, copied 
-below. If we discover it's not general enough to accommodate all exposure types,
-we need to revise the format in coworking. 
-
-.. list-table:: Constants 
+.. list-table:: ID Table 
 	:widths: 10, 5, 15
 	:header-rows: 1
 
-	* - Constant
-	  - Value
-	  - Note
-	* - 
+	* - Component
+	  - ME_ID
+	  - Notes
+	* - Mean exposure
+	  - 2547
 	  - 
+	* - Standard deviation
+	  - 15788
 	  - 
+	* - Relative risk
+	  - 9030
+	  - Must be accessed with get_draws
```

Comment:
I have the same question as BMI for what these relative risks refer to

Reply:
These relative risks refer to the risk-outcome pairs listed above; when pulling relative risks data using get_draws() it returns all causes linked to that risk. I'm merging in this PR for now, but can go back and clarify later if needed!